xwiki-platform-core/xwiki-platform-store/xwiki-platform-store-filesystem-oldcore/src/main/java/org/xwiki/store/filesystem/internal/DefaultTemporaryAttachmentSessionsManager.java

```diff
@@ -119,16 +113,24 @@ public XWikiAttachment uploadAttachment(DocumentReference documentReference, Par
             // document since it's a temporary attachment, but it is still useful to have a minimal knowledge of the
             // document it is stored for.
             xWikiAttachment.setDoc(new XWikiDocument(documentReference, documentReference.getLocale()), false);
+            this.attachmentValidator.validateAttachment(xWikiAttachment.getLongSize(), () -> {
+                try {
+                    return Optional.of(xWikiAttachment.getContentInputStream(this.contextProvider.get()));
+                } catch (XWikiException e) {
+                    this.logger.warn("Failed to get the input stream for attachment [{}]. Cause: [{}]", xWikiAttachment,
```

Comment:
I find it very strange to basically ignore this problem, I would expect something that serious to fail the validation with a cause.